docs/UserGuide.md

```diff
@@ -262,6 +262,24 @@ _Details coming soon ..._
 
 ## Command summary
 
+<<<<<<< HEAD
+| Action         | Format, Examples                                                                                                           |
+|----------------|----------------------------------------------------------------------------------------------------------------------------|
+| **Add**        | `add n/COMPANY a/ADDRESS`<br> e.g., `add n/KFC a/Yishun Street 81`                                                         |
+| **Clear**      | `clear` followed by `confirm`                                                                                              |
+ | **Create**     | `create n/NAME coy/COMPANY_NAME p/NUMBER e/EMAIL`<br> e.g., `create n/James coy/MacDonalds p/82692192 e/James@example.com` |
+| **Delete**     | `delete INDEX`<br> e.g., `delete 3`                                                                                        |
+| **Edit**       | Coming Soon                                                                                                                |
+| **Exit**       | `exit`                                                                                                                     |
+| **Find**       | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find MacDonalds`                                                                 |
+| **Filter**     | Coming Soon                                                                                                                |
+| **List**       | `list`                                                                                                                     |
+| **Help**       | `help` or `help [COMMAND]` <br> e.g.,`help` or `help add` or `help sort`                                                   |
+| **Sort**       | Coming Soon                                                                                                                |
+| **Buy**        | `buy INDEX g/GOODS q/QUANTITY price/PRICE` <br/> e.g `buy 2 g/apples q/100 price/1.5`                                      |
+| **Sell**       | `sell INDEX g/GOODS q/QUANTITY price/PRICE` <br/> e.g `sell 2 g/apples q/100 price/1.5`                                    |
+| **User Guide** | `user_guide`                                                                                                               |
+=======
```

Comment:
There might be conflicts yet to be resolved. Please resolve them.

Reply:
Okay will do thank you :)